lib/games_manager.rb

```diff
@@ -92,43 +83,20 @@ def get_season_results(team_id)
     @games.each do |game|
       if team_id == game.away_team_id || team_id == game.home_team_id
         summary[game.season] = [] if summary[game.season].nil?
-        if team_id == game.home_team_id && game.home_goals > game.away_goals
-          result = "W"
-        elsif team_id == game.away_team_id && game.home_goals < game.away_goals
-          result = "W"
-        elsif game.home_goals == game.away_goals
-          result = "T"
-        else
-          result = "L"
-        end
+        result = "non-win"
+        result = "W" if (team_id == game.home_team_id && game.home_goals > game.away_goals) || (team_id == game.away_team_id && game.home_goals < game.away_goals)
```

Comment:
we're going to get dinged on having lines that are too long with this one

Reply:
maybe assign game away/home goals/team id to shorter variables?